### PR TITLE
fix(RHINENG-10194): Allow managing and exporting descriptions in poli…

### PR DESCRIPTION
**Pull request by jiridostal**
Allows to display the Description column in the table. This is mostly done for the purpose of exporting, there is not so much space for the description in the table, so some widths needed to be readjusted.

Description column will be hidden by default.

### CODE REVIEW

File: src/PresentationalComponents/PoliciesTable/Columns.js [hunk 1]
```diff
@@ -97,4 +97,5 @@ export default [
   Systems,
   BusinessObjective,
   ComplianceThreshold,
+  Description,
```
[the remark this answers is not in the document]
Comment (bastilian) [reply src/PresentationalComponents/PoliciesTable/Columns.js:100]: Oh, I didn't realise we want to also optionally show it in the actual table.

[the remark this answers is not in the document]
Comment (jiridostal) [reply src/PresentationalComponents/PoliciesTable/Columns.js:100]: Well, tried that, but was not successful (column not added to the export), I think 'exportableColumns' needs to be a subset of 'columns'

**Comment by bastilian** [comment]
@jiridostal The `description` column does appear in the table when selected, but it doesn't seem to be included in the export yet. I can take a look and see what we'd need to change to make it work.

**Comment by bastilian** [comment]
@jiridostal Ah, my bad! I thought it was meant to be in the export in any case not only if it is selected as a column.

**Review by bastilian — APPROVED** [review]
The only thing left now is fixing the cypress tests. It probably  just fails because there is now a new column! Thank you @jiridostal!

**Pull request merged**